setup.py

```diff
@@ -46,6 +46,7 @@
             "pre-commit",
             "pytest-asyncio",
             "pytest>=6.1.1",
+            "mock",
```

Comment:
what's the reason for this change and the change in test_groupchat.py? Can we remove them?

Reply:
sorry. I was not using the remote git. I fixed it now.

Reply:
This file is still modified.

Reply:
fixed.